app/api/show-api/src/main/java/com/example/show/service/dto/response/TicketingAlertReservationServiceResponse.java

```diff
@@ -18,35 +14,24 @@ public record TicketingAlertReservationServiceResponse(
 
     public static TicketingAlertReservationServiceResponse as(
         LocalDateTime ticketingAt,
-        List<TicketingAlert> reservedAlerts,
-        LocalDateTime now
+        List<TicketingAlert> reservedAlerts
     ) {
-        Set<TicketingAlertTimeApiType> distinctAlertTimes = reservedAlerts.stream()
+        List<TicketingAlertReservationTimeServiceParam> times = reservedAlerts.stream()
             .map(alert -> {
-                int minutesGap = (int) Duration.between(alert.getAlertTime(), ticketingAt).toMinutes();
-                return TicketingAlertTimeApiType.getTicketingAlertTimeByMinutesGap(minutesGap);
-            }).collect(Collectors.toSet());
+                int beforeMinutes = (int) Duration.between(alert.getAlertTime(), ticketingAt)
+                    .toMinutes();
```

Comment:
리뷰 포인트 1
다음과 같이 long을 int로 변환하면 더 안전한 변환이 될 것 같습니다.

```suggestion
                int beforeMinutes = Long.valueOf(Duration.between(alert.getAlertTime(), ticketingAt).toMinutes()).intValue();
```